backend/server/auth.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
         breed: req.body.breed,
       },
     }).then((result) => {
+      console.log(result);
```

Comment:
Though it won't matter since we're essentially done, I would suggest removing console logs from front-end code